### ISSUE TITLE
Feature Request: Add a Keybind to ask Follow-up Questions

### ISSUE THREAD

**Issue by dangero2000** [cartertemm/AI-content-describer#9]
The AI Content Describer usually does a pretty good job at identifying things for me. However, there are some occasions when I need more details about something. It would be great if I could press a key to ask the AI a follow-up question so I can get the extra details I need, much like Be My AI does.

**Comment by cartertemm** [comment]
This is planned after the upcoming release, which will be a total refactor of the add-on and it's features. Once that is stable, I will implement your suggestion and post updates here.

I appreciate the feedback!

**Comment by cartertemm** [comment]
> The feature is available in the multimodal-input branch

If you don't feel comfortable building the addon (instructions in readme.md), here is a pre-release build that you can download:

https://ctemm.me/AIContentDescriber_multimodal-beta.nvda-addon

**Comment by dangero2000** [comment]
From my testing thus far, this works pretty well. I had some connection errors a couple times, but that's probably just a fluke. I have occasional connection drops. Other than that, it works flawlessly.

**Comment by cartertemm** [comment]
The latest version containing this functionality is out in the add-on store, so closing as completed.

If at any point you find bugs or have feature suggestions, feel free to open another ticket and I will attend to them. Sorry again for the wait on this.

Thank you!
